configs/fts-benchmarks.json

```diff
@@ -0,0 +1,109 @@
+        },
+        "searches": [
+          {
+            "sub": "a",
+            "description": "Single term all fields",
+            "dataset": "datasets/search_terms.csv",
+            "concurrent_clients": 50,
+            "command": "FT.SEARCH rd0 \"__field:term__\""
+          },
+          {
+            "sub": "b",
+            "description": "Single term specific field",
+            "dataset": "datasets/search_terms.csv",
+            "concurrent_clients": 50,
+            "command": "FT.SEARCH rd0 \"@field1:__field:term__\""
+          },
+          {
+            "sub": "c",
+            "description": "Proximity all fields",
+            "dataset": "datasets/proximity_phrases.csv",
+            "concurrent_clients": 50,
+            "command": "FT.SEARCH rd0 \"__field:phrase__\""
+          },
+          {
+            "sub": "d",
+            "description": "Proximity specific inorder",
+            "dataset": "datasets/proximity_phrases.csv",
+            "concurrent_clients": 50,
+            "command": "FT.SEARCH rd0 \"@field1:__field:phrase__\""
+          },
+          {
+            "sub": "e",
+            "description": "Mixed pattern field-specific",
+            "dataset": "datasets/mixed_match.csv",
+            "concurrent_clients": 50,
+            "command": "FT.SEARCH rd0 \"@field1:__field:phrase__\""
+          },
+          {
+            "sub": "f",
+            "description": "Single term all fields + NOCONTENT",
+            "dataset": "datasets/search_terms.csv",
+            "concurrent_clients": 50,
+            "command": "FT.SEARCH rd0 \"__field:term__\" NOCONTENT"
+          },
+          {
+            "sub": "g",
+            "description": "Single term specific field + NOCONTENT",
+            "dataset": "datasets/search_terms.csv",
+            "concurrent_clients": 50,
+            "command": "FT.SEARCH rd0 \"@field1:__field:term__\" NOCONTENT"
+          },
+          {
+            "sub": "h",
+            "description": "Proximity all fields + NOCONTENT",
+            "dataset": "datasets/proximity_phrases.csv",
+            "concurrent_clients": 50,
+            "command": "FT.SEARCH rd0 \"__field:phrase__\" NOCONTENT"
+          },
+          {
+            "sub": "i",
+            "description": "Proximity specific inorder + NOCONTENT",
+            "dataset": "datasets/proximity_phrases.csv",
+            "concurrent_clients": 50,
+            "command": "FT.SEARCH rd0 \"@field1:__field:phrase__\" NOCONTENT"
+          },
+          {
+            "sub": "j",
+            "description": "Mixed pattern field-specific + NOCONTENT",
+            "dataset": "datasets/mixed_match.csv",
+            "concurrent_clients": 50,
+            "command": "FT.SEARCH rd0 \"@field1:__field:phrase__\" NOCONTENT"
+          }
+        ]
+      }
+    ],
+    "standalone_ingestion": [],
```

Comment:
Why is it called "standalone_ingestion"?

Wouldn't standalone vs CME apply to the entire benchmark including ingestion and search?

Also, how do we plan to add support for CME here?